recipes/seekr2_openmm_plugin/meta.yaml

```
@@ -0,0 +1,98 @@
+{% set name = "seekr2_openmm_plugin" %}
+{% set version = "v0.1.1" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/seekrcentral/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 84f237433c1b56deb9eb56937fc72847d0bdf799c4bff2fea71b09f2c5db69ca
+
+# used command: shasum -a 256 seekr2_openmm_plugin-0.1.0.tar.gz
+
+build:
+  number: {{ build }}
```

Keep this as a plain number, otherwise this will lead to issues with the bots.
```suggestion
  number: 0
```